### ISSUE TITLE
Support for 2018.x Intellij IDEA versions.

### ISSUE THREAD

**Issue by romanmikhailov** [HaxeFoundation/intellij-haxe#789]
After update to latest Intellij IDEA version (2018.1) all haxe modules are treated as unknown.

```
Unknown Module Type
Cannot determine module type for the following modules:

"app" (type 'HAXE_MODULE')
All mentioned modules will be treated as Unknown modules.
```

```
IntelliJ IDEA 2018.1 (Ultimate Edition)
Build #IU-181.4203.550, built on March 26, 2018
JRE: 1.8.0_152-release-1136-b20 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.12.6
```

**Comment by EricBishton** [comment]
The current plugin is marked for 2017.x releases.  I will create a new point build today, and upload it if it passes the smoke test.  (There are usually compatibility changes between the releases.)

**Comment by EricBishton** [comment]
Fixed with the 1.0.1 release.
